.github/workflows/main.yml

```diff
@@ -1521,6 +1521,48 @@ jobs:
           paths: "./*/*/build/test/*/results/TEST-*.xml"
 
 
+  java-debugger-test:
+    name: Java Debugger ${{ matrix.config }} on Linux/JDK ${{ matrix.java }}
+    # equals env.test_java == 'true'
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'Java') || contains(github.event.pull_request.labels.*.name, 'ci:all-tests') || github.event_name != 'pull_request' }}
+    needs: base-build
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    strategy:
+      matrix:
+        java: [ '11', '17' ]
+      fail-fast: false
+    steps:
+
+      - name: Set up JDK ${{ matrix.java }}
+        uses: actions/setup-java@v3
+        with:
+          java-version: ${{ matrix.java }}
+          distribution: ${{ env.default_java_distribution }}
+
+      - name: Setup Xvfb
+        run: |
+          echo "DISPLAY=:99.0" >> $GITHUB_ENV
+          Xvfb :99 -screen 0 1024x768x24 > /dev/null 2>&1 &
+
+      - name: Download Build
+        uses: actions/download-artifact@v3
+        with:
+          name: build
+
+      - name: Extract
+        run: tar --zstd -xf build.tar.zst
+
+      - name: debugger.jpda ${{ matrix.config }}
+        run: ant $OPTS -Dtest.config=${{ matrix.config }} -f java/debugger.jpda test
```

Comment:
you sure you want to set this`-Dtest.config=${{ matrix.config }}`? This isn't actually set, so the variable would be empty.